kotlinpoet/src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -245,6 +245,19 @@ public class FunSpec private constructor(
     return null
   }
 
+  private fun CodeBlock.returnsWithoutLinebreak(): CodeBlock {
+    val originCodeBlockBuilder = toBuilder()
```

Comment:
Is there a way to optimize the code to only call `toBuilder()` if there is something to replace?

Reply:
I think so, but I'm ready to go to bed now. I'll change it tomorrow morning (I'm at UTC+8 time zone).

Reply:
I've changed it so that the `toBuilder` is now called on demand

Reply:
However, many functions actually have a return method, this optimization may not have a significant effect

Reply:
Yeah, good point, the code looks good though, so let's leave it as is!